.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: python
+
+python:
+  - "2.6"
+  - "2.7"
+  - "3.3"
+  - "3.4"
+  - "3.5"
+  - "3.6"
+
+env:
+ - REQUESTS="requests" # latest
+ - REQUESTS="requests==2.5" # min version of requests library 
```

Comment:
using `>=2.5` I would assume would pull the latest version however I want to travis to test with both the min version 2.5 and the latest version.